src/robust.jl

```diff
@@ -0,0 +1,98 @@
+# Robust Statistics
+
+#############################
+#
+#   Trimming outliers
+#
+#############################
+
+# Trimmed set
+"""
+    trim(x, p=0.2)
+
+Remove a proportion `p` of its highest elements, and `p` of its lowest elements and return the result. If you want to compute the variance of `mean(trim(x,p))`, use `trimvar`.
```

Comment:
Break lines at 92 chars. "its" isn't explicit: say "vector `x`" instead. Mention that the result is sorted. I would also Change the last sentence into `To compute the trimmed mean of 'x', use...; to compute the variance of this quantity, use...`.